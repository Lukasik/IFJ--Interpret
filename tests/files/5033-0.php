<?php

$else="aaaaaaaaa\\\$\\\$\t\t\x30\x31\x24aaaaaaaaa";
$z=put_string($else);