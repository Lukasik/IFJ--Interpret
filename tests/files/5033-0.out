aaaaaaaaa\$\$		01$aaaaaaaaa